src/components/inventory/GeneralInfo.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
-import { useStore } from 'react-redux';
+import { useStore, useSelector } from 'react-redux';
 import GeneralInformation from '@redhat-cloud-services/frontend-components-inventory-general-info/cjs';
 
 const GeneralInformationTab = () => {
-    return <GeneralInformation store={useStore()} />;
+    const writePermissions  = useSelector(
+        ({ permissionsReducer: { writePermissions } }) => writePermissions
+    );
```

Comment:
It might be better to use safe accessor here
```suggestion
  const writePermissions = useSelector(
    ({ permissionsReducer }) => permissionsReducer?.writePermissions
  );
```